students/joshua-bone/lesson3/slicing.py

```diff
@@ -0,0 +1,37 @@
+#Joshua Bone - UW Python 210 - Lesson 3
+#12/9/2018
+#Slicing Lab Exercise
+
+def exchange_first_last(seq):
+  return seq[-1:] + seq[1:-1] + seq[0:1]
+
+def remove_every_other(seq):
+  return seq[::2]
+
+def mid_last_first(seq):
+  l = len(seq)
+  a, b, c = seq[0:l//3], seq[l//3:2*l//3], seq[2*l//3:]
```

Comment:
since a, b, c don't provide much more context, you can just return:
`return seq[0:l//3] + seq[l//3:2*l//3] + seq[2*l//3:]`